cmd/mesh/manifest.go

```diff
@@ -89,6 +93,27 @@ func genManifests(inFilename string, setOverlayYAML string) (name.ManifestMap, e
 	if err != nil {
 		return nil, err
 	}
+	// for installation path contains version number only
+	isp := mergedICPS.InstallPackagePath
```

Comment:
should check that at most one of isp.InstallPackagePath or isp.Version are set

Reply:
shouldn't this be done at validation package?

Reply:
yes, that's a good idea. let's move it to validation later.